hw7/hw7_task4.py

```diff
@@ -0,0 +1,14 @@
+def min_max(new_list: list):
+    """
+    Implement your own implementation of function max and min
+    """
+    for i in range(len(new_list) - 1):
+        for j in range(len(new_list) - i - 1):
+            if new_list[j] > new_list[j + 1]:
+                new_list[j], new_list[j + 1] = new_list[j + 1], new_list[j]
+    return new_list[0], new_list[-1]
```

Comment:
let it be. but it is too complicated. 2 nested loops are not a good approach. 
another solution just sort a list with already defined function)